packages/react-components/react-datepicker-compat/src/components/DatePicker/useDatePickerStyles.ts

```diff
@@ -1,75 +1,65 @@
-import { tokens } from '@fluentui/react-theme';
-import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { makeStyles, makeResetStyles, mergeClasses } from '@griffel/react';
```

Comment:
_nit:_ Alphabetical order

```suggestion
import { makeResetStyles, makeStyles, mergeClasses } from '@griffel/react';
```